SalesforceSDK/Universal/Pages/AccountPage.xaml.cs

```diff
@@ -88,7 +88,18 @@ public ObservableCollection<ServerSetting> Servers
         protected override void OnNavigatedTo(NavigationEventArgs e)
         {
             base.OnNavigatedTo(e);
-            SetupAccountPage();
+            CheckIfLoginNeeded();
+        }
+
+        private void CheckIfLoginNeeded()
+        {
+            Account account = AccountManager.GetAccount();
+            if (account == null)
+            {
+                SDKServiceLocator.Get<ILoggingService>().Log("Account object is null, calling StartLoginFlowAsync",
```

Comment:
should you update this log statement?  It says it's calling StartLoginFlowAsync but it's not directly calling that here.  Updating this would help when debugging via logs later on